src/Facade/ResourceBuilders/WorkstationElementsResourceBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Linn.Stores2.Facade.ResourceBuilders
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Resources;
+    using Linn.Stores2.Domain.LinnApps.Stores;
+    using Linn.Stores2.Resources.Stores;
+
+    public class WorkstationElementsResourceBuilder : IBuilder<WorkstationElement>
+    {
+        public WorkstationElementResource Build(WorkstationElement model, IEnumerable<string> claims)
+        {
+            var claimsList = claims?.ToList() ?? new List<string>();
+
+            return new WorkstationElementResource
+            {
+                WorkstationElementId = model.WorkstationElementId,
+                WorkstationCode = model.WorkstationCode,
+                PalletNumber = model.PalletNumber,
+                LocationId = model.LocationId,
+                DateCreated = model.DateCreated.ToString("o"),
+                CreatedBy = model.CreatedBy?.Id,
+                CreatedByName = model.CreatedBy?.Name
+            };
+        }
+
+        public string GetLocation(WorkstationElement model)
+        {
+            return $"/stores2/work-stations/{model.WorkstationCode}/{model.WorkstationElementId}";
+        }
+
+        object IBuilder<WorkstationElement>.Build(WorkstationElement entity, IEnumerable<string> claims) =>
+            this.Build(entity, claims);
+
+        private IEnumerable<LinkResource> BuildLinks(WorkstationElement model, IEnumerable<string> claims)
```

Comment:
you aren't calling this method in the from the .Build() - might as well just do so